### ISSUE TITLE
How to install mods?

### ISSUE THREAD

**Issue by fantoro** [yvt/openspades#677]
Hey, so I thought about installing a mod and I found one, do I put it in `/usr/share/games/openspades/Resources` or `$HOME/.local/share/openspades/Resources`? Also should I make a directory in Resources (So that I have to put the files in `Resources/[ModName]`?

**Comment by NotAFile** [comment]
Have a look at the wiki

https://github.com/yvt/openspades/wiki/Resource-loading-order
https://github.com/yvt/openspades/wiki/User-Resource-Directory

**Comment by feikname** [comment]
Also, as per the [XDG spec](https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html):

> XDG_DATA_HOME defines the base directory relative to which user specific data files should be stored. If $XDG_DATA_HOME is either not set or empty, a default equal to $HOME/.local/share should be used.

I think XDG_DATA_HOME is empty by default in Ubuntu systems. I'm quite sure this was mentioned in the wiki before, I'll take a look.

**Comment by kmsiapps** [comment]
I think we need a wiki page explaining how to install mods. It's hard for newcomers to know about it..

**Comment by feikname** [comment]
@kmsiapps Agreed, the information on mods is quite scarce (and if I recall correctly, some is outdated). Perhaps I will take a look into this tomorrow.

**Comment by NotAFile** [comment]
we also need to actually link to it in visible places and make a guide on *creating* good mods aka .pak files

**Comment by fantoro** [comment]
To be honest I could try to make an easy to use GUI ModManager for OS but:
I don't know whether I should use GTK+ or Qt.
I can't cross compile and statically link the default c++ libraries (At least for mingw, I've never tried making a Mac OS app), although I am dualbooted with Windows so I can always compile there (even though compiling with libraries on Windows is a mess for me for some reason).

Also I guess that my coding/programming skills suck (Mostly at making a gui and networking and well, most of advanced stuff) because I never made anything good in C++ except  tutorials and simple cli programs.

**Comment by NotAFile** [comment]
it might be ideal to just add it as a tab in the OpenSpades launcher...

**Comment by kmsiapps** [comment]
I'm making a wiki page(https://github.com/yvt/openspades/wiki/Applying-packaged-mods)
Please see and help :)

**Comment by feikname** [comment]
@kmsiapps I added links to your newly created pages in the [Modding](https://github.com/yvt/openspades/wiki/Modding) section of the wiki 😉.

Hopefully, the information there now should be enough to get started at least.

**Comment by Tacki0n4** [comment]
The information is very hard to find. Im currently new to this game and im trying to add a sniper scope
